OasisDEX/oasis-borrow
# Move marketing pages to Borrow

# [Move marketing pages to Borrow](https://app.clubhouse.io/oazo-apps/story/2117/moving-marketing-pages-to-borrow)

<please insert a clubhouse link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>

- Imported pages: privacy, support, about, careers, careers/[slug] (the terms page was already imported).
- Added some entries to the theme file.
- Updated footer links.
  
## How to test 🧪
  <Please explain how to test your changes>

- Check out the pages.
- In order to see the careers page with mocked careers, revert this commit locally: 4bf77bb
    
## Definition of done ✔️
